gen/definitions/properties.yaml

```diff
@@ -2895,3 +2896,74 @@ infraRsAccBndlSubgrp:
       # Response Status Code: 400, Error Code: 182, Error Message: Validation failed PortBlk has a reln to AccBndlSubgrp not contained by AccBndlGrp its parent is associated to.
       target_dn: ""
 
+macsecIfPol:
+  overwrites:
+    admin_st: admin_state
+
+macsecRsToKeyChainPol:
+  targets:
+    - class_name: "macsecKeyChainPol"
+      target_dn: "uni/infra/macsecpcont/keychainp-macsec_key_chain_0"
+      relation_resource_name: "macsec_key_chain"
+      properties:
+        name: "macsec_key_chain_0"
+    - class_name: "macsecKeyChainPol"
+      target_dn: "uni/infra/macsecpcont/keychainp-macsec_key_chain_1"
+      relation_resource_name: "macsec_key_chain"
+      properties:
+        name: "macsec_key_chain_1"
+
+macsecRsToParamPol:
+  targets:
+    - class_name: "macsecParamPol"
+      target_dn: "uni/infra/macsecpcont/paramp-macsec_parameters_0"
+      relation_resource_name: "macsec_parameters"
+      properties:
+        name: "macsec_parameters_0"
+
+macsecParamPol:
+  ignores:
+    - "sakExpiryTime"
+  overwrites:
+    conf_offset: confidentiality_offset
+    key_svr_prio: key_server_priority
+    sec_policy: security_policy
+    replay_window: window_size
+  documentation:
+    cipherSuite: "The cipher suite of the %s object."
+    confOffset: "The confidentiality offset in bytes of the %s object."
+    secPolicy: "The security policy of the %s object."
+    keySvrPrio: "The key server priority of the %s object. The value must be in the range 0 - 255."
+    replayWindow: "The replay protection window size of the %s object. The value must be in the range 0 - 4294967295."
+  test_values:
+    all:
+      confidentiality_offset: "offset-50"
+      key_server_priority: "39"
+      window_size: "200"
+
+macsecKeyChainPol:
+  default_values:
+    parent_dn: "uni/infra/macsecpcont"
+
+macsecKeyPol:
+  remove_valid_values:
+    endTime:
+      - "infinite"
+    startTime:
+      - "now"
+  documentation:
+    keyName:  "The key name of the %s object."
+    endTime: "The end time of the %s object. The default value cannot be used directly, since APIC will calculate this value to the datetime representation which results in a applied state that does not match the planned state. Do not provide this attribute when APIC should calculate the current datetime ('now'). Furthermore, make sure that the time-zone part (in the full example this is '-08:00') matches the time-zone configured on APIC. Not doing so will result in unexpected behaviour."
```

Comment:
```suggestion
    endTime: "The end time of the %s object. The default value cannot be used directly, since APIC will calculate this value to the datetime representation which results in a applied state that does not match the planned state. Do not provide this attribute when APIC should calculate the current datetime ('now'). Furthermore, make sure that the time-zone part (in the full example this is '-08:00') matches the time-zone configured on APIC. Providing a different time-zone will result in unexpected behaviour."
```